SQLiteStudio3/guiSQLiteStudio/translations/guiSQLiteStudio_zh_CN.ts

```diff
@@ -558,7 +558,8 @@
     <message>
       <location filename="../constraints/columngeneratedpanel.ui" line="71"/>
       <source>Named constraint:</source>
-      <translation>已命名的约束：</translation>
```

Comment:
This doesn't seem right. The actual comment is moved into the <translatorcomment> tag, while the translation is reversed to the English version. Is this by a mistake?